### ISSUE TITLE
This extension doesn't work with custom process template

### ISSUE THREAD

**Issue by Gabriele2** [konradsikorski/VSS.Extension.ChangeState#5]
Could you please make available this really useful extension for custom process templates too? Could you please suggest how modify the source code in order to achieve this?

**Comment by konradsikorski** [comment]
I will try to implement this in next release. Unfortunately TFS API doesn't allow me to access custom template data so user will have to configure extension manually in new configuration dialog.

**Comment by Gabriele2** [comment]
Hi Konrad,

anyway it will be great solution. I'll use your extension in a really great IT project.

Thanks!


Best Regards


Gabriele Di Antonio


Sent from Outlook<http://aka.ms/weboutlook>
________________________________

**Comment by nschonni** [comment]
Not sure if you've started the work on this one yet, but I had an idea on how to implement this.
You could allow a free-form field(s) somewhere to allow users to upload 1 or 2 (`stateFlow`/`stateToReason`) JSON objects to represent their process flows like in https://github.com/konradsikorski/VSS.Extension.ChangeState/blob/master/src/statesDetails.js

**Comment by konradsikorski** [comment]
Yes I have the same idea :D 
I have version 2 ready for publishing and could work on this feature but the VSS API has a bug which is blocking me now :(

**Comment by nschonni** [comment]
Do you have a link to the VSS bug so we can go upvote it?

**Comment by konradsikorski** [comment]
Yes, here it is: https://github.com/Microsoft/vss-web-extension-sdk/issues/68
And thx for your help
